### PR TITLE
DashboardScene: Adjust failing scenes e2e

### PR DESCRIPTION
**Pull request by Sergej-Vlasov**
Adjust failing scenes e2e tests

### CODE REVIEW

File: e2e/scenes/dashboard-suite/set-options-from-ui.spec.ts [hunk 1]
```diff
@@ -110,19 +116,21 @@ describe('Variables - Set options from ui', () => {
...
       .next()
+      .should('have.text', 'AAA')
+      .next()
       .should('have.text', 'AAB')
       .next()
       .should('have.text', 'AAC');
   });
 
-  it('removing a value that is part of dependents options should remove the new values dependant options', () => {
+  it.only('removing a value that is part of dependents options should remove the new values dependant options', () => {
```
Comment (axelavargas) [inline e2e/scenes/dashboard-suite/set-options-from-ui.spec.ts:133]: Should we remove the `.only`?

Comment (Sergej-Vlasov) [reply e2e/scenes/dashboard-suite/set-options-from-ui.spec.ts:133]: yep, good catch! Thank you!

**Comment by Sergej-Vlasov** [comment]
workflow run with lates changes ✅ 
https://github.com/grafana/grafana/actions/runs/9017667036/job/24776587725?pr=87553

**Review by axelavargas — APPROVED** [review]
🥳 great work @Sergej-Vlasov

**Pull request merged**